Dockerfile

```diff
@@ -9,7 +9,9 @@ ENV METEOR_PACKAGE_DIRS=/MATScommon/meteor_packages
 # Assume we're passed the repo root as build context
 COPY apps/${APPNAME}/package*.json ${APP_SOURCE_FOLDER}/
 
-RUN bash ${SCRIPTS_FOLDER}/build-app-npm-dependencies.sh
+#RUN bash ${SCRIPTS_FOLDER}/build-app-npm-dependencies.sh
```

Comment:
Glad this worked. A small nit - we should clean up this commented-out line before we merge this PR to keep the Dockerfile clean.